lua/acf/server/sv_acfbase.lua

```diff
@@ -135,11 +135,16 @@ function ACF_CalcDamage( Entity , Energy , FrAera , Angle )
 	local HitRes = {}
 
 
-	
+	if istable(Entity) == false then -- ACF feeds us a table for some reason when we shoot living things.

```

Comment:
This is a bug in itself probably. Would be a good idea to fix it instead of patching it here.